angles/include/angles/angles.h

```diff
@@ -215,6 +290,13 @@ namespace angles
   static inline bool shortest_angular_distance_with_limits(double from, double to, double left_limit, double right_limit, double &shortest_angle)
   {
 
+    // If either left_limit or right_limit is outside the "standard" unit circle,
+    // call shortest_angular_distance_with_large_limits instead
+    if(std::fabs(left_limit) > M_PI || std::fabs(right_limit) > M_PI)
+    {
+      return shortest_angular_distance_with_large_limits(from, to, left_limit, right_limit, shortest_angle);
```

Comment:
With the description and parallel implementation I was surprised to see this here. I'm not sure that it makes sense for behavior to change if I slowly increase the limit value past a magic number. In particular this I think will yield quite different answers if you are doing operations near the back of the unit circle and you're leveraging the wrap around the back. Suddenly your check will return false if you shift your interval across the back of the unit circle.

I think that it might make more sense to keep them separate and clarify that the existing one is computing on a unit circle and that all input values would be normalized onto the [-M_PI, M_PI) range.

And the logic for a multiple turn joint is kept separate in your new computation.

Reply:
I understand. I think my mistake was in assuming that `shortest_angular_distance_with_limits` would be used with `left_limit` and `right_limit` strictly inside the `[-pi,pi)` range. I will remove this change and update the description of `shortest_angular_distance_with_large_limits`.